views/styles/chart_styles.js

```diff
@@ -118,4 +118,29 @@ export const chartStyles = (theme) => ({
     WebkitAppearance: 'none',
     outlineOffset: 'none',
   },
+  tooltipContainer: {
+    width: '200px',
+    padding: '14px 14px 7px 14px ',
+    background: colors.transparentWhite,
+  },
+  tooltipHeaderRow: {
+    display: 'flex',
+    width: '200px',
```

Comment:
can this be 100%?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/191/commits/4ed7b45e0dfa83017cab82c474a3512ffc0c38c2